runtime/doc/options.txt

```diff
@@ -5560,7 +5560,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	old Vi scripts.
 
 						*'renderoptions'* *'rop'*
-'renderoptions' 'rop'	Removed. {Nvim} will choose the best renderer available.
+'renderoptions' 'rop'	Removed. {Nvim}
```

Comment:
> Perhaps we can just replace vi_diff.txt

Let's leave vi_diff.txt alone for now, but create a new `vim_diff.txt` file. 

> then tack on (See |nvim-diff-options|)

I suggest `|vim-differences|` in analogy with the existing `vi-differences` tag.

> nvim_diff.txt

Technically it should be `vim_diff.txt` :)

> I can convert this PR,

:+1: